### PR TITLE
Fix issues in Part 4

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**
- Readme code using incorrect properties and API methods.
- XAML end tag in MainPage.xaml was incorrect.
- Shorter LINQ Query using MinBy

### CODE REVIEW

**Comment by khalidabuhakmeh** [comment]
This is kind of related but noticed the details page looks off for me.

Screenshot 2022-05-26 at 10 50 48

**Review by jamesmontemagno — APPROVED** [review]

File: Part 4 - Platform Features/README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ We can easily check to see if our user is connected to the internet with the bui
 
     ```csharp
     builder.Services.AddSingleton<IConnectivity>(Connectivity.Current);
-    builder.Services.AddSingleton<IGeolocation>(Geolocation.Default);
-    builder.Services.AddSingleton<IMap>(Map.Default);
+    builder.Services.AddSingleton<IGeolocation>(Geolocation.Current);
+    builder.Services.AddSingleton<IMap>(Map.Current);
```
Comment (jamesmontemagno) [inline Part 4 - Platform Features/README.md:29]: They are both .Default for me..  https://github.com/dotnet/maui/blob/main/src/Essentials/src/Geolocation/Geolocation.shared.cs#L37

Current is private

File: Part 4 - Platform Features/README.md [hunk 2]
```diff
@@ -157,7 +156,7 @@ This project is pre-configured with all required permissions and features needed
     {
         try
         {
-            await map.OpenAsync(Monkey.Latitude, Monkey.Longitude, new MapLaunchOptions
```
Comment (jamesmontemagno) [inline Part 4 - Platform Features/README.md:160]: This api was changed back. Make sure you are compiling against the latest GA build

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]
Jon fixed up the mainpage.xaml... for the other stuff bysides minby would need to be reverted and then updated in the source code for 5, 6, and final

**Comment by khalidabuhakmeh** [comment]
I'm running `sudo dotnet workload update` but the CLI only ever finds the `RC1` bits. 😢

**Pull request closed without merging**